DataFormats/HcalRecHit/interface/HBHERecHit.h

```diff
@@ -22,6 +22,9 @@ class HBHERecHit : public CaloRecHit {
   /// get the id
   inline HcalDetId id() const { return HcalDetId(detid()); }
 
+  inline void setChiSquared(const float chi2) {chiSquared_ = chi2;}
+  inline float getChiSquared() const {return chiSquared_;}
```

Comment:
... better yet, use chi2 in the method names to match what we have in track and ecal hit classes.